dm/dm-best-practices.md

```
@@ -5,7 +5,11 @@ summary: 了解使用 TiDB Data Migration (DM) 进行数据迁移的一些最佳
 
 # DM 数据迁移最佳实践
 
+<<<<<<< HEAD
```

markdownlint-cli2 (0.23.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)